packages/web3-eth-abi/src/types.ts

```diff
@@ -0,0 +1,57 @@
+export interface JsonAbiParameter {
```

Comment:
will be easy for future developers if you add reference of source definition/doc for these types

Reply:
All the references will be added to next PR. As these structs been modified there.